cogs/Responses/mentions.json

```diff
@@ -14,6 +14,11 @@
     "I'm not bad I'm just coded that way.",
     "Code me like one of your optimization algorithms.",
     "Ask <@337266376941240320> instead.",
-    "They say all the best things in life are free, so give all your <:feesh:427018890137174016> and your code to me!"
+    "They say all the best things in life are free, so give all your <:feesh:427018890137174016> and your code to me!",
+    "Who are you and what do you want?!",
+    "Psssst, wanna know a secret? Xplio smells!",
```

Comment:
maybe use lower case xplio? he never uses Xplio

Reply:
Xplio just made sense since it's the start of a sentence, I thought about just xploio a lot